### PR TITLE
fix: defer some sensitive @Value resolution

### PR DESCRIPTION
**Pull request by jhaeyaert**
**Issue**

https://gravitee.atlassian.net/browse/ARCHI-299

**Description**

This PR defers most of the `@Value` and some bean instantiations in order to make them eligible for secret injection (waiting for a proper global solution).

**Additional context**

This mainly covers the server bootstraps that are susceptible to being configured using secrets such as keystore passwords.
- Jetty server
- Vertx HTTP/TCP server
- Management Server

---
**Gravitee.io Automatic Deployment**

🚀 A prerelease version of this package has been published on Gravitee's private artifactory, you can:
 - use it directly by updating your project with version: `4.6.1-archi-299-defer-resolve-values-SNAPSHOT`
 - download it from Artifactory [here](https://odbxikk7vo-artifactory.services.clever-cloud.com/gravitee-snapshots/io/gravitee/node/gravitee-node/4.6.1-archi-299-defer-resolve-values-SNAPSHOT/gravitee-node-4.6.1-archi-299-defer-resolve-values-SNAPSHOT.zip)

### CODE REVIEW

File: gravitee-node-jetty/src/main/java/io/gravitee/node/jetty/JettyHttpConfiguration.java [hunk 1]
```diff
@@ -15,266 +15,112 @@
...
 package io.gravitee.node.jetty;
 
-import org.springframework.beans.factory.annotation.Value;
-import org.springframework.context.annotation.Configuration;
+import io.gravitee.node.api.configuration.Configuration;
 
 /**
  * @author David BRASSELY (david.brassely at graviteesource.com)
  * @author GraviteeSource Team
  */
```
Comment (benoitgravitee) [inline gravitee-node-jetty/src/main/java/io/gravitee/node/jetty/JettyHttpConfiguration.java:23]: @RequiredArgsConstructor ?

File: gravitee-node-jetty/src/main/java/io/gravitee/node/jetty/JettyHttpConfiguration.java [hunk 2]
```diff
@@ -15,266 +15,112 @@
...
     public boolean isAccessLogEnabled() {
-        return accessLogEnabled;
-    }
-
-    public void setAccessLogEnabled(boolean accessLogEnabled) {
-        this.accessLogEnabled = accessLogEnabled;
+        return configuration.getProperty("jetty.accesslog.enabled", Boolean.class, true);
     }
 
     public String getAccessLogPath() {
```
Comment (benoitgravitee) [inline gravitee-node-jetty/src/main/java/io/gravitee/node/jetty/JettyHttpConfiguration.java:80]: I suppose this configuration was done using the setter, right ?

File: gravitee-node-management/src/main/java/io/gravitee/node/management/http/vertx/configuration/HttpServerConfiguration.java [hunk 3]
```diff
@@ -15,193 +15,87 @@
...
 package io.gravitee.node.management.http.vertx.configuration;
 
+import io.gravitee.node.api.configuration.Configuration;
 import io.vertx.core.http.HttpServerOptions;
-import org.springframework.beans.factory.annotation.Value;
 
 /**
  * @author David BRASSELY (david.brassely at graviteesource.com)
  * @author GraviteeSource Team
  */
```
Comment (benoitgravitee) [inline gravitee-node-management/src/main/java/io/gravitee/node/management/http/vertx/configuration/HttpServerConfiguration.java:24]: @RequiredArgsCOnstructor ?

File: gravitee-node-jetty/src/main/java/io/gravitee/node/jetty/JettyHttpConfiguration.java (hunk 2 above)
Comment (jhaeyaert) [reply gravitee-node-jetty/src/main/java/io/gravitee/node/jetty/JettyHttpConfiguration.java:80]: I just turned:

```java
    @Value("${jetty.accesslog.path:${gravitee.home}/logs/gravitee_accesslog_yyyy_mm_dd.log}")
    private String accessLogPath;
```

into the corresponding `configuration.getProperty(...) `

**Review by ytvnr — APPROVED** [review]

**Pull request merged**